deployment/packer/cac_packer.py

```diff
@@ -13,7 +13,7 @@ class CacStackException(Exception):
     pass
 
 
-def get_ubuntu_ami(region, creds):
```

Comment:
These weren't being used to retrieve the base AMI because we just pull down a public text file with Ubuntu AMIs IDs.